dht_test.go

```diff
@@ -1581,7 +1587,7 @@ func TestHandleRemotePeerProtocolChanges(t *testing.T) {
 	// dhtB becomes a server
 	require.NoError(t, dhtB.setMode(modeServer))
 
-	// which means dhtA should have it in the RT again
+	// which means dhtA should have it in the RT again because of fixLowPeers
```

Comment:
This is always true because `fixLowPeers` will add connected peers to the RT and we are connected to the peer who just started supporting the DHT protocol again.

Reply:
makes sense